workflow/docker_util.py

```diff
@@ -38,6 +38,40 @@ def prepare_path_docker(orig_path: PurePath) -> str:
     return prepared_path
 
 
+def is_container_running(container_name: str) -> bool:
+    """check if docker container is running 
+
+    Args:
+        container_name (str): name of container
+
+    Returns:
+        bool
+    """
+
+    client = docker.from_env()
+    
+    # test if current Container (TPS or Cytoscape) is running
+    try:
+        container = client.containers.get(container_name)
+
+    # the current container is not running (cant be found) -> False
+    except docker.errors.NotFound as ex:
+        print(f"{ex.explanation}")
+        return False
+
+    # APIErrors can happen
+    except docker.errors.APIError as ex:

```

Comment:
So we get an `APIError` every time we try to `get` a container that isn't already running?  If that is the normal and expected behavior, I don't think we need to print this error message.